proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -597,7 +617,7 @@ ProToken plusMinusStart(int inputType) throws IOException {
         case 'z':
           append();
           getChar();
-          ttype = ProParserTokenTypes.FILENAME;
+          ttype = PreprocessorParser.FILENAME;
           break;
         case '.':
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1151)